### ISSUE TITLE
Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by ImGrumpy** [plugCubed/plugAPI#150]
After recently updating the API, I started receiving this error...  Looks like it happens when the last DJ leaves the waitlist.
```
C:\bot\node_modules\plugapi\lib\room.js:195
        for (let i = 0; i < ids.length; i++) {
                               ^
TypeError: Cannot read property 'length' of undefined
```

**Comment by thedark1337** [comment]
Known bug and fixed by https://github.com/plugCubed/plugAPI/pull/140. ATM I am trying to figure out the playlist functionality as well as fixing auto reconnect before I release a new version of v5. Sorry you're getting the bug, I had to rush the release for v5 due to the role updates and did not have the proper amount of time to bug test things

**Comment by ImGrumpy** [comment]
Got it... I read that but didn't realize that doing an npm update wouldn't pick up that fix.

**Comment by thedark1337** [comment]
Issue should be resolved with the latest release v5.1.0, please let me know if it is not.
